datastore/google/cloud/datastore/entity.py

```diff
@@ -183,6 +183,19 @@ def kind(self):
         if self.key:
             return self.key.kind
 
+    @property
+    def id(self):
+        """Get the id of the current entity.
+
+        .. note::
+          This relies entirely on the :class:`gcloud.datastore.key.Key`
```

Comment:
1. There is no "Sphinx / rST lint" but I've seen @lukesneeringer use 4 space indents after a `.. note::` and IMO it's good form to leave an empty line between the ``.. note::` directive and the actual content.
1. There is no such thing a `gcloud` any more, not sure where this text comes from.
1. Please use ID (not "id")

Reply:
I will follow up with the documentation changes for both on a separate PR.  It would be inconsistent to change one and not the other and it would detract from this PR if I sent it together.

Reply:
Just change this part and feel free to send a follow-up PR to update the other part.

Reply:
See #4641.  I changed them both there.